TestAssets/TestProjects/KitchenSink/TestLibrary/TestLibrary.csproj

```diff
@@ -4,6 +4,12 @@
     <TargetFramework>netstandard1.5</TargetFramework>
     <ProjectGuid>{BB691360-49BC-46EB-8BA9-6A69A54CD4B0}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Instead of setting the `DocumentationFile` property based on the configuration, just set the `GenerateDocumentationFile` to true.